client/src/components/Landing.js

```diff
@@ -6,14 +6,31 @@ import { makeStyles } from '@material-ui/styles';
 import Register from './Register';
 import Login from './Login';
 
+import bg from '../assets/bg.jpg';
+
 const useStyles = makeStyles({
   root: {
     height: '100vh',
-    width: '100%'
+    width: '100%',
+    margin: -16
   },
   splash: {
     backgroundImage: 'linear-gradient(#498FEE, #88BAFE)',
-    color: 'white'
+    color: 'white',
+    height: '100%',
+    width: '100%',
+    opacity: 0.92,
+    margin: 0
+  },
+  background: {
+    backgroundImage: `url(${bg})`,
+    backgroundSize: 'cover',
+    height: '101%',
```

Comment:
is this supposed to be 101% as opposed to 100%

Reply:
I had some issues with the default margins on material-ui components and this was my hacky solution to make that component stretch to the bottom of the page. Perhaps we could discuss alternatives for future iterations.